plugins/woocommerce/includes/class-wc-privacy.php

```diff
@@ -139,6 +139,7 @@ public function queue_cleanup_personal_data() {
 		self::$background_process->push_to_queue( array( 'task' => 'trash_pending_orders' ) );
 		self::$background_process->push_to_queue( array( 'task' => 'trash_failed_orders' ) );
 		self::$background_process->push_to_queue( array( 'task' => 'trash_cancelled_orders' ) );
+		self::$background_process->push_to_queue( array( 'task' => 'anonymize_orders_query' ) );
```

Comment:
I think this should be `anonymize_refunded_orders`, otherwise the function won't get called.